src/plugins/mf_priority.cpp

```diff
@@ -98,11 +126,27 @@ int64_t priority_calculation (flux_plugin_t *p,
         return -1;
     }
 
+    auto it = released_jobs.find (jobid);
+    if (it != released_jobs.end ()) {
+        // job has an "age" associated with it; fetch the timestamp
+        t_released = it->second;
+        // get current time
+        t_current = std::chrono::duration_cast <std::chrono::duration<double>>
```

Comment:
Then if those are time_point, you can directly subtract the system_clock::now result instead of doing this duration_cast at all.  If you need the double in age_factor then use the cast on the result of the subtraction.